tests/client/client.rs

```diff
@@ -0,0 +1,21 @@
+use bson::Bson;
+use mongodb::client::MongoClient;
+
+#[test]
+fn list_databases() {
```

Comment:
Can you add more tests for getting a cursor for `list_databases`?
Test that you can get a cursor with `results < batch size, results > batchSize, and results == batchSize`